content/components/selectpanel.mdx

```diff
@@ -64,58 +166,264 @@ Select panel should be anchored to its trigger.
     <img
       width="464"
       alt="Select panel with a back button, close button, Heading, subheading, five multiple choice options with three of them selected, and apply and cancel buttons."
-      src="https://user-images.githubusercontent.com/40274682/228910121-cfcac62f-c672-48ba-97e4-c851e1f65c49.png"
+      src="https://github.com/primer/design/assets/980622/71915284-9deb-4acf-9f41-bd2e6f8a58e3"
     />
-    <Caption>If there are more than single options in a menu, have a user confirm their choices.</Caption>
+    <Caption>
+      For multi selection have the user confirm their selection unless an <a href="#cancelsubmit">exception is made</a>.
```

Comment:
We should not be allowing exceptions for multi-select. If someone is choosing to incorporate a pattern that will fail WCAG, we definitely shouldn't be advertising that this is happening.

Reply:
Fixed